Sectors/Lines - LTC.txt

```diff
@@ -64,7 +66,6 @@ DISPLAY:London TC NW:London TC NW:London TC NE
 DISPLAY:London TC SW:London TC NW:London TC NE
 DISPLAY:London TC SE:London TC NW:London TC NE
 DISPLAY:London AC Daventry:London TC NW:London TC NE
-DISPLAY:London TC East:London TC NW:London TC NE
```

Comment:
Think this should stay as it is, this line informed TC E that TC NE/W are split (see the other times it's displayed), it's for info rather than to split the sectors or show a handoff.

Reply:
Does TC East need to know that though? I could understand AC Daventry finding it useful to know that as it could have handoffs to both NW and NE when covering the full Daventry and Clacton bandbox position but TCE will never have a handoff to TCNW unless covering TCNE top down for some reason. Similarly, AC Clacton does not have that line visible at any point. AC Dover is similar to Clacton in that it never has handoffs to TCNW and TCNE Worthing does have handoffs to both when covering TCSW topdown.

Happy to make the change as long as it is definitely what we want?

Reply:
OK, let's then remove it.